lib/osf-components/addon/components/file-browser/file-rename-modal/template.hbs

```diff
@@ -0,0 +1,50 @@
+<OsfDialog
+    @isOpen={{@isOpen}}
+    @onClose={{@onClose}}
+    @closeOnOutsideClick={{true}}
+    data-test-file-rename-modal
+    as |dialog|
+>
+    <dialog.heading data-test-rename-heading>
+        {{t 'osf-components.file-browser.file_rename_modal.heading'}}
+    </dialog.heading>
+    <dialog.main>
+        <div local-class='RenameInput'>
+            <Input
+                data-test-user-input
+                aria-label={{t 'osf-components.file-browser.file_rename_modal.input_aria'}}
+                @value={{this.newFileName}}
+                @type='text'
+            />
+        </div>
+    </dialog.main>
+    <dialog.footer>
+        <Button
+            data-test-cancel-button
+            data-analytics-name='Rename Cancel'
+            @type='secondary'
+            {{on 'click' (
+                queue
+                this.resetFileNameValue
+                dialog.close
+            )
+            }}
+        >
+            {{t 'osf-components.file-browser.file_rename_modal.cancel'}}
+        </Button>
+        <Button
+            data-test-disabled-rename
+            data-analytics-name='Rename Save'
+            @type='create'
+            disabled={{not this.isValid}}
```

Comment:
Should this check `this.updateFileName.isRunning`?

Reply:
disabling it while the task is running will prevent users from sending multiple rename requests at once. Preventing that behavior will avoid confusing errors and toast messages from spamming the users

Reply:
It should disable the button when the task is running to prevent triggering multiple rename tasks.